packages/core/src/components/Review/Review.jsx

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReviewLink from './ReviewLink';
+import classNames from 'classnames';
+
+/*
+`<Review>`
+
+@react-component Review
+
+Style guide: components.review.react
+*/
+
+/**
+ * The `Review` component is used for listing a summary of information entered
+ * by a user. Its content includes a heading, value, and edit link.
+ */
+export class Review extends React.PureComponent {
+  heading() {
+    if (this.props.heading) {
+      return (
+        <h3 className="ds-c-review__heading ds-text ds-u-margin-bottom--0 ds-u-font-weight--bold ds-u-display--inline-block">
```

Comment:
Will we need a way to customize the heading level used here? This is something @1Copenut did for the `MonthPicker` component by adding a `headingLevel` prop.